tests/functional/test_cli.py

```diff
@@ -50,17 +50,19 @@ def tearDown(self):
         shutil.rmtree(self.artifacts_dir)
 
     @parameterized.expand([
-        ("request_through_stdin"),
-        ("request_through_argument")
+        ("request_through_stdin", lambda_builders_protocol_version),
```

Comment:
can you add a test that fails compat check?

Reply:
yes the test is present below.

Reply:
https://github.com/awslabs/aws-lambda-builders/pull/78/files#diff-736a032bebd5a17dd2e352d0b55f76acR115